.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: php
 
 php:
-  - 5.5
```

Comment:
Unfortunately, need to remove support for 5.5, due to the use of the variadic functions